src/sentry/deletions/defaults/platform_external_issue.py

```diff
@@ -0,0 +1,7 @@
+from ..base import ModelDeletionTask
+
+
+class PlatformExternalIssueDeletionTask(ModelDeletionTask):
+    def mark_deletion_in_progress(self, instance_list):
+        # No status to track this.
+        pass
```

Comment:
Do you need these methods? The base implementation should skip over models that don't have a `status` field.

Reply:
The problem is some models *do* have status, but they don't track `ObjectStatus`.

Reply:
I don't see a status column on `PlatformExternalIssue` though.

Reply:
If someone wishes to add status and schedule deletions for external issues, they will hopefully explicitly add testing, and, fortunately, see the explicit configuration, adjusting both at the same time.